src/main.rs

```diff
@@ -1,45 +1,31 @@
 use axum::Server;
 use clap::Parser;
-use routes::build_routes;
 use std::{net::SocketAddr, panic, path::PathBuf, sync::Arc};
 use tracing::{error, info, subscriber};
 use tracing_subscriber::EnvFilter;
 
-use crate::state::{Db, State};
-
-mod client;
-mod error;
-mod routes;
-mod signature;
-mod state;
-mod util;
-
-pub use error::{Error, Result};
+use actiserve::{
+    base_url,
+    config::Config,
+    routes::build_routes,
+    state::{Db, State},
+};
 
 // TODO: move this to Args
 const PORT: u16 = 4242;
```

Comment:
do we still need this?

Reply:
nope! The should be getting pulled from the config as well